### PR TITLE
client error report

### LINKED ISSUE #12: Add client-side bug reporting
## Description
- As the maintainer of experiment infrastructure, I want to know ASAP if something is not working, so that I can fix it quickly, ensure that I am collecting valid data, and avoid wasting money. 
- As a participant, I want to easily and quickly report problems and frustrations so that I can have them resolved for my benefit and the benefit of other participants, without creating more problems or frustrations.

Ideally, we'd have an unobtrusive button in the top corner that says something like "report a problem". When you click it, it would capture the state of the game and necessary debugging information, and it would also give the player a chance to describe the problem with text, and to do a screen capture to show the problem.

There is an argument for making this survey not a part of Empirica itself, as it is most useful when Empirica breaks, and we wouldn't want to have a single point of failure. It would be awesome if it could continue to function even if the Empirica display crashes entirely, and also if there is a backend problem that causes data not to be recorded from the client. There may be some third-party tool that Empirica could load up in the client browser prior to loading anything else, and which would capture and aggregate data to a third-party server. This could either be an open-source project that we run a separate server to manage, or a truly independently managed SAAS. 

We are already using sentry to monitor client side errors, I expect there is a way we can add this button using their platform.

### PR DESCRIPTION
**Pull request by kepstein23**
I have not been able to test that the stack trace is actually being sent to Sentry yet.

I was also not able to figure out how to allow users to upload a screenshot, so I'm still looking for a way to do that.

closes #12

### CODE REVIEW

**Comment by kepstein23** [comment]
Need to change "Submit crash process" to "Submit", remove default text, and remove email validation

File: client/src/App.jsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import { usePlayer } from "@empirica/player";
 import TopicSurvey from "./intro-exit/Surveys/gov_reduce_income_inequality";
 import team_viability from "./intro-exit/Surveys/team_viability";
 import quality_control from "./intro-exit/Surveys/quality_control";
+import * as Sentry from '@sentry/browser';
```
Comment (JamesPHoughton) [inline client/src/App.jsx:15]: Is this import actually used in this file?

File: client/src/main.jsx [hunk 2]
```diff
@@ -1,28 +1,79 @@
...
+              eventId: Sentry.captureEvent({
+                message: 'User Feedback',
+                stacktrace: [
+                  Sentry.captureMessage
+                ],
+              }),
+              subtitle: "Please let us know what went wrong below.",
+              subtitle2: '',
+              labelName: "MTurk ID",
+              labelEmail: "What stage in the experiment were you in?",
```
Comment (JamesPHoughton) [inline client/src/main.jsx:60]: If the email validation is still in place, we can ask them instead to just ignore this field

**Review by JamesPHoughton — CHANGES_REQUESTED** [review]

File: client/src/App.jsx (hunk 1 above)
Comment (kepstein23) [reply client/src/App.jsx:15]: yes, we need it to have the dialog show up when our report button is clicked since it's not in the Sentry.init

**Pull request merged**